packages/gateway/src/features/certificate/root-resolvers.test.ts

```diff
@@ -156,7 +156,7 @@ describe('Certificate root resolvers', () => {
           authHeaderRegister
         )
       ).rejects.toThrowError(
-        'Create certificate is only allowed for natlsysadmin'
+        'Create or update certificate is only allowed for natlsysadmin'
```

Comment:
Notice that here the `expect(` statement isn't awaited or returned either so effectively this test doesn't work as intented

Reply:
I have returned now! Thanks